lexis360/parser.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Lexis 360
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  let param = parsedUrl.query || {};
+  let rawUrl = ec.url;
```

Comment:
It may be easier to parse the raw URL instead of using the plain string, to avoid side effects.
```javascript
  const URL = require('url');
  parsedUrl = URL.parse(ec.url, true); // true = parse query string
```